### PR TITLE
Register logger for updates

### PR DESCRIPTION
**Pull request by jbout**
Requires https://github.com/oat-sa/generis/pull/200

### CODE REVIEW

File: models/classes/extension/UpdateLogger.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    {
+        $service = $this->getServiceLocator()->get(FileSystemService::SERVICE_ID);
+        $filesystem = $service->getFileSystem($this->getOption(self::OPTION_FILESYSTEM));
+        
+        $updateId = 'update_'.time();
+        while ($filesystem->has($updateId.'.log')) {
+            $count = isset($count) ? $count + 1 : 0;
+            $updateId = 'update_'.time().'_'.$count;
+        }
+        echo $updateId;
```
Comment (siwane) [inline models/classes/extension/UpdateLogger.php]: Can you remove it please

File: scripts/update/Updater.php [hunk 2]
```diff
@@ -504,6 +506,22 @@ public function update($initialVersion) {
         }
         
         $this->skip('2.22.0', '4.3.1');
+        
+        if ($this->isVersion('4.3.1')) {
+            try {
+                $this->getServiceManager()->get(UpdateLogger::SERVICE_ID);
```
Comment (siwane) [inline scripts/update/Updater.php]: why not has?

File: models/classes/extension/UpdateLogger.php (hunk 1 above)
Comment (siwane) [inline models/classes/extension/UpdateLogger.php]: remove it please

File: manifest.php [hunk 3]
```diff
@@ -78,7 +80,8 @@
             dirname(__FILE__).'/scripts/install/setServiceState.php',
             dirname(__FILE__).'/scripts/install/setJsConfig.php',
             dirname(__FILE__).'/scripts/install/registerEntryPoint.php',
-            dirname(__FILE__).'/scripts/install/setLocaleNumbersConfig.php'
+            dirname(__FILE__).'/scripts/install/setLocaleNumbersConfig.php',
+            AddLogFs::class
```
Comment (siwane) [inline manifest.php]: not found

**Comment by jbout** [comment]
remerged develop

**Pull request merged**